Makefile

```diff
@@ -178,9 +179,11 @@ install-pam: $(PAM_MODULE)
 	install -d $(DESTDIR)$(PAM_MODULE_DIR)
 	install $(PAM_MODULE) $(DESTDIR)$(PAM_MODULE_DIR)
 
+ifdef $(PAM_CONFIG_DIR)
```

Comment:
It needs to be `ifdef PAM_CONFIG_DIR`.  Otherwise the file won't be installed even if PAM_CONFIG_DIR is defined.

Reply:
Ohh, good point. Guess I was mislead by the example in the GNU Make documentation which has `ifdef $(foo)`. But the text clearly states that `$(foo)` is expanded and the result is check for being defined. Sorry about that. :/